src/messages.ts

```diff
@@ -509,11 +509,11 @@ export const completeOptionsMessages = defineMessages(
   'ui.src.completeOptions',
   {
     essentialLabel: {
-      defaultMessage: 'Essential purposes',
+      defaultMessage: 'Essential',
```

Comment:
these arent being displayed anywhere anymore, right? why the edits to them?

Reply:
they are still used as defaults, although now every purpose is supposed to have a title and description. should i remove these and the tests associated with default values?

Reply:
tests need to be cleaned up for sure, I think some of them use these to compare against